docs/faas/faas-spans.md

```diff
@@ -199,6 +203,18 @@ which the invoked FaaS instance reports about itself, if it's instrumented.
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+This span represents outgoing call to FaaS service.
+
+The values reported by the client for the attributes listed below SHOULD be equal to
+the corresponding [FaaS resource attributes][] and [Cloud resource attributes][],
+which the invoked FaaS instance reports about itself, if it's instrumented.
```

Comment:
just checking you intended to leave this text above also?

Reply:
another great catch, thanks!